docs/README.md

```diff
@@ -61,6 +61,8 @@ Both `conseil` and `lorre` write to `syslog`. The logs are verbose enough to det
 
 `conseil` and `lorre` use [HOCON](https://github.com/lightbend/config/blob/master/HOCON.md) format to store configuration. Read more at [Lightbend Config](https://github.com/lightbend/config) github page.
 
+For additional extra fine-grained configuration you can refer to [this appendix](extra-custom-config.md)
```

Comment:
I'm generally not sure why we have a duplicate readme here?

Reply:
I don't get it as well. It looks like one is more technical and detailed and the second one is more in "what's this project about" style.